packages/material-ui/src/Hidden/HiddenCss.js

```diff
@@ -1,42 +1,85 @@
 import * as React from 'react';
+import clsx from 'clsx';
 import PropTypes from 'prop-types';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
 import capitalize from '../utils/capitalize';
-import withStyles from '../styles/withStyles';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import useTheme from '../styles/useTheme';
+import { getHiddenCssUtilityClass } from './hiddenCssClasses';
 
-const styles = (theme) => {
+const sizes = ['xl', 'lg', 'md', 'sm', 'xs'];
```

Comment:
We cannot rely on only the default breakpoints. I believe this is why the test is broken. We need to support dynamically added values too.

Reply:
Although it wasn't the cause of the failing test (assuming you're referring to the custom breakpoints test), I realize that support for custom breakpoints was pretty much completely removed.  I've tweaked the code to support dynamic breakpoints again!